recipes/sdl2/bld.bat

```diff
@@ -0,0 +1,40 @@
+:: MSVC library for Python 2.7 is not found automaticaly if it is 
+:: installed in the user's AppData folder. Add it to PATH
+setlocal EnableDelayedExpansion
+
+set TARGET=Release
+:: Set target back to Debug for 64-bits builds with VS2008 and VS2010
+if %VS_MAJOR% LSS 14 (
+	if %ARCH% == 64 (
+		set TARGET=Debug
+	)
+)
+
+:: When installing Visual C++ compiler tools for Python 2.7, all files are placed in the users AppData folder
+:: conda-bld tries to call vcvarsall at its classic location, which of course fails, so we have to call it from its
+:: new location in the the users appdata folder manually, but only if this folder exists of course.
+if "%PY_VER%"=="2.7" (

```

Comment:
Yes, this is also necessary for appveyor, as the VC tools are also installed in a user folder. If you look at the appveyor log of this package when built for Python 2.7 (https://ci.appveyor.com/project/conda-forge/staged-recipes/build/1.0.7155/job/h1invbjev7vrekql) you'll see:

~~~
(root) C:\Miniconda-x64\conda-bld\work\SDL2-2.0.4>call "C:\Program Files (x86)\Microsoft Visual Studio 9.0\Common7\Tools\..\..\VC\vcvarsall.bat" amd64 
The specified configuration type is missing.  The tools for the
configuration might not be installed.
~~~

around line 350 indicating that vcvarsall cannot be found at this location. Later at line 361 you see

~~~
-- Check for working C compiler: C:/Users/appveyor/AppData/Local/Programs/Common/Microsoft/Visual C++ for Python/9.0/VC/bin/amd64/cl.exe
-- Check for working C compiler: C:/Users/appveyor/AppData/Local/Programs/Common/Microsoft/Visual C++ for Python/9.0/VC/bin/amd64/cl.exe -- works
~~~
which means that, also on appveyor, the VC tools are found in the users appdata folder. conda-build does not check this location by default, hence I've added this snippet to my build scripts which guards for this.